doc/api/tracing.md

```diff
@@ -36,7 +36,7 @@ The available categories are:
 
 By default the `node`, `node.async_hooks`, and `v8` categories are enabled.
 
-```txt
+```text
```

Comment:
```suggestion
```bash
```

Reply:
I think `shell` would be preferable here because it's agnostic (vs bash/zsh/sh/fish).

Reply:
I don't believe there is a way to highlight an agnostic shell. The relevant PRs are https://github.com/nodejs/node/pull/33510 and https://github.com/nodejs/node/pull/33486 where it was decided that `shell` was too obscure of an info string as it had often been confused for `console`. I agree that some commands are agnostic, but for this we've opted to use `bash` however unfortunate that may be. There is an alternative choice of `powershell` if that makes more sense.